### ISSUE TITLE
how to collect the indicator of training results???

### ISSUE THREAD

**Issue by cleveryg** [kubeflow/katib#931]
/kind bug

how to collect the indicator of training results???
**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]

   After bayesianoptimization automatic training is completed, katib cannot collect the training indicator result. Katib UI always shows the training status as running. Could you please tell me how to collect the training indicator and update the training indicator status on katib UI???
  NAME                                                      READY   STATUS      RESTARTS   AGE
bayseopt-example-66xhrlbn-bzhsl                           0/1     Completed   0          55m
bayseopt-example-67qc55c6-ctkgs                           0/1     Completed   0          55m
bayseopt-example-bayesianoptimization-76cbd855f9-4rs7d    1/1     Running     0          56m
bayseopt-example-bltbmcv2-gg722                           0/1     Completed   0          55m
bayseopt-example-pbk45nvp-xdxjc                           0/1     Completed   0          55m
bayseopt-example-rqjjqpnv-8mfss                           0/1     Completed   0          55m
bayseopt-example-wdd8k5zn-wqz25                           0/1     Completed   0          55m
bayseopt-example5-82t4rn6n-mxg8n                          0/1     Completed   0          18m
bayseopt-example5-9zs2b7k5-mldxn                          0/1     Completed   0          18m
bayseopt-example5-bayesianoptimization-7b9b9455bd-gjqjb   1/1     Running     0          19m
bayseopt-example5-bcl7h9xc-ptbx9                          0/1     Completed   0          18m
bayseopt-example5-j2l7xqc5-swftk                          0/1     Completed   0          18m
bayseopt-example5-ldm9pxxb-qvpl9                          0/1     Completed   0          18m
bayseopt-example5-q8hfwsx8-4c5sq                          0/1     Completed   0          18m

My yaml file is as follows:
apiVersion: "kubeflow.org/v1alpha3"
kind: Experiment
metadata:
  namespace: kubeflow
  labels:
    controller-tools.k8s.io: "1.0"
  name: bayesianoptimization-example
spec:
  objective:
    type: maximize
    goal: 0.99
    objectiveMetricName: Validation-accuracy
    additionalMetricNames:
      - accuracy
  algorithm:
    algorithmName: bayesianoptimization
    algorithmSettings:
      - name: "random_state"
        value: "10"
  parallelTrialCount: 3
  maxTrialCount: 12
  maxFailedTrialCount: 3
  MetricsCollectorSpec:
    Collector:
      Kind: stdOut
  parameters:
    - name: --lr
      parameterType: double
      feasibleSpace:
        min: "0.01"
        max: "0.03"
    - name: --num-layers
      parameterType: int
      feasibleSpace:
        min: "2"
        max: "5"
    - name: --optimizer
      parameterType: categorical
      feasibleSpace:
        list:
        - sgd
        - adam
        - ftrl
  trialTemplate:
    goTemplate:
        rawTemplate: |-
          apiVersion: batch/v1
          kind: Job
          metadata:
            name: {{.Trial}}
            namespace: {{.NameSpace}}
          spec:
            template:
              spec:
                containers:
                - name: {{.Trial}}
                  image: docker.io/katib/mxnet-mnist-example
                  command:
                  - "python"
                  - "/mxnet/example/image-classification/train_mnist.py"
                  - "--batch-size=64"
                  {{- with .HyperParameters}}
                  {{- range .}}
                  - "{{.Name}}={{.Value}}"
                  {{- end}}
                  {{- end}}
                restartPolicy: Never

**What did you expect to happen:**

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Kubeflow version:
- Minikube version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

**Comment by cleveryg** [comment]
kubeflow 0.7.0
kubenrnetes 1.15.4
OS:CentOS Linux release 7.7.1908 (Core)

**Comment by gaocegege** [comment]
dup with #930
